configure

```diff
@@ -0,0 +1,338 @@
+#!/bin/sh
+### Custom configure script of Dinit.
+## Initial prepartion
+set -eu
+cd "$(dirname "$0")"
+. build/version.conf
```

Comment:
Don't include a makefile fragment into a shell script. I don't think it's needed anyway?